### ISSUE TITLE
Timeouts can no longer be disabled via option exist:set-timeout

### ISSUE THREAD

**Issue by wolfgangmm** [eXist-db/exist#2529]
As documented, the default query timeout in eXist could be set to infinite by adding

```xquery
declare option exist:timeout "-1";
```

It looks like support for this was mistakenly dropped in b7cc1140c0397f9c8be68afa8917e2e176c0c2ef and should be restored.

**Comment by dizzzz** [comment]
are you sure about that hashcode?

**Comment by adamretter** [comment]
@wolfgangmm I think you linked the wrong commit?

**Comment by joewiz** [comment]
@wolfgangmm I think you meant https://github.com/eXist-db/exist/commit/ee77d49025512fc86d0979e8ef43882e793c266c.

**Comment by dizzzz** [comment]
when looking at the code.... it should be ok...

**Comment by dizzzz** [comment]
<img width="708" alt="image" src="https://user-images.githubusercontent.com/1700062/57856314-1385c880-77ed-11e9-9a10-5a3c0d210f92.png">

are we sure it ever worked?

**Comment by duncdrum** [comment]
@dizzzz yes i can confirmed that it used to work in 3.x. Will test 4.x, and 5.x
